src/app/Services/InitService.ts

```diff
@@ -114,7 +114,8 @@ export class InitService {
         this.messageService.resetVariables();
         this.cacheService.reset();
         this.messageService.me = null;
-        localStorage.clear();
+        // Don't clear it to save theme settings.
```

Comment:
If someone signs in another account, he will use the previous one's setting and may try to resend some messages.

Reply:
So, what about save the preferred theme settings to our server side?

Reply:
OK.